goodtablesio/integrations/s3/blueprint.py

```diff
@@ -170,3 +96,204 @@ def create_job():
     _run_validation(bucket, job_id)
 
     return jsonify({'job_id': job_id})
+
+
+# API
+
+# TODO:
+# it should be synced with general
+# approach we use for API (see api blueprint)
+
+
+@s3.route('/api/bucket/<bucket_id>')
```

Comment:
I agree - I've added it to further steps here - https://github.com/frictionlessdata/goodtables.io/issues/158 (also review how our API works as one system)